scripts/updates/github-projects.txt

```diff
@@ -327,6 +327,8 @@ vifm|vifm/vifm
 virustotal-cli|VirusTotal/vt-cli
 vis|martanne/vis
 vtutils|fornwall/vtutils
+vulkan-headers|KhronosGroup/Vulkan-Headers|\d+\.\d+\.\d+
+vulkan-tools|KhronosGroup/Vulkan-Tools|\d+\.\d+\.\d+
```

Comment:
These packages do not have GitHub Releases and cannot be added to automatic updates. GitHub API doesn't provide way to fetch the latest tag but only latest release.

See file header.

Reply:
Here is what will be:
```
~ $ curl --silent --location "https://api.github.com/repos/KhronosGroup/Vulkan-Headers/releases/latest"
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/repos#get-the-latest-release"
}
```

And here example of package with releases (truncated due to long output):
```
~ $ curl --silent --location "https://api.github.com/repos/fornwall/vtutils/releases/latest"
{
  "url": "https://api.github.com/repos/fornwall/vtutils/releases/11566552",
  "assets_url": "https://api.github.com/repos/fornwall/vtutils/releases/11566552/assets",
  "upload_url": "https://uploads.github.com/repos/fornwall/vtutils/releases/11566552/assets{?name,label}",
  "html_url": "https://github.com/fornwall/vtutils/releases/tag/v0.5",
  "id": 11566552,

  <truncated>

  "node_id": "MDc6UmVsZWFzZTExNTY2NTUy",
  "tag_name": "v0.5",
  "target_commitish": "master",
  "name": "v0.5",
  "draft": false,
  "prerelease": false,
  "created_at": "2015-06-21T11:23:44Z",
  "published_at": "2018-06-20T11:32:30Z",

  <truncated>

  "tarball_url": "https://api.github.com/repos/fornwall/vtutils/tarball/v0.5",
  "zipball_url": "https://api.github.com/repos/fornwall/vtutils/zipball/v0.5",
  "body": ""
}
```

Reply:
TIL thanks